src/Elsa.Dashboard/Authorization/Constants.cs

```diff
@@ -0,0 +1,15 @@
+namespace Elsa.Dashboard.Authorization
+{
+  public class Constants
+  {
+    public static class AppRole
```

Comment:
Are we intending to have nested classes here?  Not opposed to it - just have never seen it done before like this.